CSharpMath/Atom/LaTeXParser.cs

```diff
@@ -500,7 +500,7 @@ private static void MathListToLaTeX
           }
           if (atom.FontStyle != outerFontStyle) {
             // open a new font style
-            builder.Append(@"\").Append(LaTeXSettings.FontStyles[atom.FontStyle]).Append("{");
+            builder.Append(@"\").Append(LaTeXSettings.FontStyles.SecondToFirst[atom.FontStyle]).Append("{");
```

Comment:
Now I need to think about the lookup direction every time I use the indexer. The original style seems more concise.

Reply:
It's natural to think about the lookup direction because there are two dictionaries and the type name itself refers to that! There was even an instance of ambiguity where TSecond was string in the tests, which required special disambiguation.

Reply:
Resolved as similar to above